src/Manager/CampaignDraftManager.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Mailjet\MailjetBundle\Manager;
+
+use \Mailjet\Resources;
+use \Mailjet\Response;
+use Mailjet\MailjetBundle\Client\MailjetClient;
+use Mailjet\MailjetBundle\Exception\MailjetException;
+use Mailjet\MailjetBundle\Model\CampaignDraft;
+
+/**
+ * https://dev.mailjet.com/email-api/v3/campaigndraft/
+ * CampaignDraft data. (list,view, create, update, delete,schedule,send ...)
+ */
+class CampaignDraftManager {
+
+    /**
+     * Mailjet client
+     * @var MailjetClient
+     */
+    protected $mailjet;
+
+    public function __construct(MailjetClient $mailjet) {
+        $this->mailjet = $mailjet;
+    }
+
+    /**
+     * List campaigndraft resources available for this apikey
+     * @return array
+     */
+    public function getAllCampaignDrafts(array $filters = null) {
+        $response = $this->mailjet->get(Resources::$Campaigndraft, ['filters' => $filters]);
+        if (!$response->success()) {
```

Comment:
As mentioned in Slack, I implemented it this way to keep the code uniform to the rest of the project. It would probably make sense that we later create a Error(management) class and use in throughout the project. I feel it's best to do that with another PR, and keep this one for the mentioned functionality.
@Nightbr  what do you think ?